drivers/pwm/pwm_nrfx.c

```diff
@@ -243,11 +255,8 @@ static int pwm_nrfx_set_cycles(const struct device *dev, uint32_t channel,
 static int pwm_nrfx_get_cycles_per_sec(const struct device *dev, uint32_t channel,
 				       uint64_t *cycles)
 {
-	/* TODO: Since this function might be removed, we will always return
-	 * 16MHz from this function and handle the conversion with prescaler,
-	 * etc, in the pin set function. See issue #6958.
-	 */
-	*cycles = 16ul * 1000ul * 1000ul;
+	const struct pwm_nrfx_config *config = dev->config;
+	*cycles = config->clock_freq;
```

Comment:
Please add a blank line between these two. It's strange that checkpatch did not complain about that.